src/SecureStore.CyberArkCCP/ErrorCode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace UiPath.Orchestrator.Extensions.SecureStores.CyberArkCCP
```

Comment:
We should not add error codes as it might clash with other implementations of the secure stores.
Somebody else can use same codes.